webClient/src/assets/i18n/i18n.js

```diff
@@ -30,8 +30,10 @@ const en = {
 		no_or_empty_message_received_from_client: "No Or Empty Message Received from Client",
 		empty_reply_from_server: "Empty Reply from Server",
 		could_not_find_app_with_id_provided: "Could not find app with ID provided",
-		status_will_appear_here: "Status will appear here",
```

Comment:
Github says this is gone, but the raw file shows it, so that's a bit weird and something to look out for.